src/main/resources/scope-conf/mail.conf

```diff
@@ -24,4 +24,8 @@ mail {
     # Contains a whitespace separated list of servers (domain names), whose SSL certificates can be trusted. When set,
     # CA signatures are apparently ignored, and only the listed servers are trusted! Set to "*" to trust any server.
     trustedServers = ""
+
+    # whether E-Mail addresses with non-ascii-symbols are considered valid
+    # The mail server needs to support SMTPUTF8 for this to work, see https://datatracker.ietf.org/doc/html/rfc6531
+    allow-utf-8 = true
```

Comment:
default to false?

Reply:
... that would make the PR breaking. Idk

Reply:
breaking ...but only since some dev-version as i see (https://github.com/scireum/sirius-web/commit/b6e2d6cfd11252dc3eba02c7bd895945a5bf6e8c)